SCF-Session-8-Assignment/src/question/Peacock.java

```diff
@@ -0,0 +1,28 @@
+package question;
+
+/**
+ * This class is for Peacock type animals
+ * @author Jugal
+ * Dated 07/24/2019
+ */
+public class Peacock extends Bird {
+
+	/**

```

Comment:
Declare some different properties of Peacock class and initialize them in Peacock class constructor and initialize Bird and Animal class property using super().

Reply:
Done